test/test_quantized.py

```diff
@@ -485,9 +485,47 @@ def test_max_pool2d_nhwc(self, X, kernel, stride, dilation, padding):
             qa, kernel_size=_pair(kernel),
             stride=_pair(kernel if stride is None else stride),
             padding=_pair(padding), dilation=_pair(dilation))
+        print(a_hat.dequantize())
+        print(a_ref)
         self.assertEqual(a_ref, a_hat.dequantize(),
                          message="ops.quantized.max_pool2d results are off")
 
+    @given(X=hu.tensor(shapes=hu.array_shapes(min_dims=3, max_dims=4,
+                                              min_side=8, max_side=10),
+                       qparams=hu.qparams()),
+           kernel=st.sampled_from((3, 5)),
+           stride=st.sampled_from((None, 1, 2)),
+           padding=st.integers(0, 2))
+    def test_avg_pool2d(self, X, kernel, stride, padding):
```

Comment:
Need to test all fields of the API. The float version looks like this:
```python
F.avg_pool2d(input, self.kernel_size, self.stride,
                            self.padding, self.ceil_mode, self.count_include_pad, self.divisor_override)
```
Specifically, please test for ceil_mode, count_include_pad and divisor override.